src/wstool/config_yaml.py

```diff
@@ -442,11 +442,8 @@ def generate_config_yaml(config, filename, header, pretty=False,
     items = [x.get_legacy_yaml(spec, exact) for x in items]
 
     if items:
-        if pretty:
-            content += yaml.safe_dump(items, allow_unicode=True,
-                                      default_flow_style=False)
-        else:
-            content += yaml.safe_dump(items)
+        content += yaml.safe_dump(items, allow_unicode=True,
```

Comment:
This seems to change the behavior of the function. Before the change, `allow_unicode` was not explicitly set to `True` if `pretty` mode is disabled.

Reply:
That's true. But I couldn't see a reason to disallow unicode when pretty is False. Is there one?

Reply:
I'm not sure in this case, but in general, I think it's good to not change something without understanding the implications or intent as this can lead to subtle bugs. One approach is to use `git blame` and track down the author to see if he or she remembers. That being said, I think this should be a separate PR if we do want the change it since it's a separate and perhaps unnecessary change.

Reply:
You are right. I'm going to close this PR as #133 is a better fix for this in this regard.
However, would be great if somebody could actually merge these critical fixes (#133 and #134).